README.md

```diff
@@ -1,43 +1,139 @@
 
-Add and remove exercises
-Add and remove sets
-For each set, add weight and reps
-For each set, mark as finished or unfinished
+- [![React][react.js]][react-url]
+- Firebase
+- React Redux
+- Axios
+- React Router
+- EsLint and AirBnb
+- Typescript
 
+### Wireframe using Figma
 
-if first time signin
-take to profile
-auto profile with info
+### Getting Started The App build out with simple wireframes using Figma.
 
-let update the username
-age
-weight
-bmi
+<img src="https://raw.githubusercontent.com/sisay2405/Physical-fitness/feature/readme/src/assets/images/homewireframe.png"
+        alt="home page"
+             height="250" width="500">
+<img src="https://raw.githubusercontent.com/sisay2405/Physical-fitness/feature/readme/src/assets/images/signinwireframe.png"
+        alt="sign in page"
+             height="250" width="500">
+            
+<img src="https://raw.githubusercontent.com/sisay2405/Physical-fitness/feature/readme/src/assets/images/signupwireframe.png"
+        alt="sign up page"
+             height="250" width="500">
+             
+<a  align="right" href="#top">Back to top</a>
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+This is an example of how you may give instructions on setting up your project locally.
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- npm
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Installation
+
+1. Get a free account at for firebase to create your own collection and add it to your .env file to connect your API to your Data base
+2. Clone the repo
+   ```sh
+   git clone https://github.com/sisay2405/Physical-fitness
+   ```
+3. Install NPM packages
+   ```sh
+   npm install
+   ```
+4. Enter your firebase link in your `.env` file in your backend
+   ```js
+   REACT_APP_FIREBASE_API_KEY = 'ENTER YOUR API KEY';
+   REACT_APP_FIREBASE_AUTH_DOMAIN = 'ENTER YOUR FIREBASE_AUTH_DOMAIN';
+   REACT_APP_FIREBASE_PROJECT_ID = 'ENTER YOUR FIREBASE_PROJECT_ID';
+   REACT_APP_FIREBASE_STORAGE_BUCKET = 'ENTER YOUR FIREBASE_STORAGE_BUCKET';
+   REACT_APP_FIREBASE_MESSAGING_SENDER_ID = 'ENTER YOUR FIREBASE_MESSAGING_SENDER_ID';
+   REACT_APP_FIREBASE_APP_ID = 'ENTER YOUR FIREBASE_APP_ID';
+   ```
+5. npm start for START the app.
+
+<a  align="right" href="#top">Back to top</a>
+<!-- USAGE EXAMPLES -->
+
+## Usage
+
+Can be used to workout the Exercises your can create your own physical fiteness web app and share it with your community. You can:
+
+- [x] creat, customezie Exercises
+- [x] Add Exercises
+- [x] Delete Exercises
+- [x] Edit Exercises
+- [x] User Sign In/Sign Up
+- [x] To creat your Exercises you will need to sign in/sign up
+
+<a  align="right" href="#top">Back to top</a>
+<!-- CONTACT -->
+
+## Contact
+
+Sisay Areaya - sareaya@alphaworks.tech , abrehasisay04@gmail.com
+
+Project Link: <a href="https://github.com/sisay2405/phiscal-fitness-app"> Physical Fitness
+            </a>
+<a  align="right" href="#top">Back to top</a>
```

Comment:
I'd recommend wrapping all these 'Back to top' links inside of paragraphs so they show up below any existing text.